service/src/main/java/uk/nhs/adaptors/gp2gp/ehr/model/EhrFhirExtractParams.java

```diff
@@ -0,0 +1,23 @@
+package uk.nhs.adaptors.gp2gp.ehr.model;
+
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class EhrFhirExtractParams {
```

Comment:
These are the parameters for the `ehr_extract_template.mustache` so maybe just name the class `EhrExtractParameters` or `EhrExtractTemplateParameters` so the naming is consistent? FHIR is not really involved at this point.

Reply:
Ok